### DISCUSSION TITLE
automatically suggest new maintenance date

### DISCUSSION THREAD

**Discussion by BioMedGPH** [Samedis-care/samedis-care-community#40] [Shipped - Live on production]
When completing a maintenance task and planning the next maintenance date you have to navigate to a different view to lookup the maintenance interval, then calculate the next date, and then find the next date in the calendar. But if you input the Maintenance finished date of a certain task, and the maintenance interval is given for this device type and maintenance type, then the calendar could automatically calculate the next maintenance date and select that date in the calendar view. Then the user could either accept this suggested new date, or deviate and choose a different date. Would make planning tasks a lot easier!

**Comment by YvesR** [comment]
Hello, this is a good idea and easy to implement, so we are getting this into the next release.
